src/planscape/planscape/urls.py

```diff
@@ -17,13 +17,14 @@
 from django.urls import path, include
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('boundary/', include('boundary.api')),
-    path('conditions/', include('conditions.urls')),
-    path('forsys/', include('forsys.urls')),
-    path('plan/', include('plan.urls')),
-    path('projects/', include('existing_projects.urls')),
+    path('planscape-backend/admin/', admin.site.urls),
```

Comment:
I think ../interface/src/app/backend-constants.ts will need to be updated to reflect this change - can you adjust the end_point?